packages/common/src/intl/locales/ar.json

```diff
@@ -47,7 +47,12 @@
   "label.supplier": "Supplier",
   "label.total": "Total",
   "label.type": "Type",
+  "label.general": "General",
+  "label.item": "Item",
+  "label.batch": "Batch",
+  "label.price": "Price",
+  "label.log": "Log",
```

Comment:
Didn't know what to key these as. Was also thinking these are very generic terms so `generic.` came to mind.. 🤔

Reply:
Yup. There's also something there with sharing localisation files with different projects